cmd/deploy.go

```diff
@@ -195,7 +195,9 @@ func deployAirflow(path, name, wsID string, prompt bool) error {
 	if err != nil {
 		return err
 	}
-	fmt.Println("Successfully pushed Docker image to Astronomer registry. Navigate to the Astronomer UI for confirmation that your deploy was successful.")
+
+	deploymentLink := buildAstroUIDeploymentLink(name, wsID)
```

Comment:
it will take a bit to deploy should we write that in? o.o lol (time to spin up)

Reply:
updated the message